rails/benchmarks/views/posts/show.html.erb

```diff
@@ -15,11 +15,5 @@
   <%= @post.author %>
 </p>
 
-<% @post.comments.each do |comment| %>
```

Comment:
@kirs I removed the associations first just to keep the benchmarks simple. We could explore further in the future if associations are a key indicator of performance :smile_cat: